include/model/CHierarchicalResultsNormalizer.h

```diff
@@ -44,6 +45,19 @@ struct MODEL_EXPORT SNormalizer {
     //! Compute a checksum for this object.
     uint64_t checksum() const;
 
+    void debugMemoryUsage(const core::CMemoryUsage::TMemoryUsagePtr& mem) const {
+        mem->setName("SNormalizer Memory Usage");
+        core::memory_debug::dynamicSize("s_Description", s_Description, mem);
+    }
+
+    std::size_t memoryUsage() const {
+        std::size_t mem = 0;
+        mem += core::memory::dynamicSize(s_Description);
+        return mem;
+    }
+
+    std::size_t staticSize() const { return sizeof(*this); }
```

Comment:
I think we only need a `staticSize` method if we're computing memory usage via a base class pointer in which case this should be an override. So I don't think it is needed here.